state.go

```diff
@@ -181,6 +181,14 @@ func (p *Proposal) Copy() *Proposal {
 	return pp
 }
 
+type CommittedSeal struct {
+	// Signature value
+	Signature []byte
+
+	// Node that signed
+	NodeID NodeID
```

Comment:
Why do we need NodeID there?

Reply:
We talked today during the sync that we will add NodeId so that we can use bls aggregated signatures